Apps/Laboratory/src/Delegates/ILaboratoryDelegate.cs

```diff
@@ -44,5 +45,15 @@ public interface ILaboratoryDelegate
         /// <param name="bearerToken">The security token representing the authenticated user.</param>
         /// <returns>A base64 encoded PDF.</returns>
         Task<RequestResult<LaboratoryReport>> GetLabReport(Guid id, string hdid, string bearerToken);
+
+        /// <summary>
+        /// Returns the public COVID-19 test results for the given patient.
+        /// </summary>
+        /// <param name="accessToken">The connection access token.</param>
+        /// <param name="phn">The patient's Personal Health Number.</param>
+        /// <param name="dateOfBirth">The patient's date of birth.</param>
+        /// <param name="collectionDate">The date the test was collected.</param>
+        /// <returns>The COVID-19 test results result for the given patient.</returns>
+        Task<RequestResult<PHSAResult<IEnumerable<PublicCovidTestResult>>>> GetPublicTestResults(string accessToken, string phn, DateOnly dateOfBirth, DateOnly collectionDate);
```

Comment:
I see your passing in the accessToken which stays consistent in this code but are more "modern" code directly looks it up via the http context.

Reply:
This pattern was borrowed from the vaccine status delegate, which authorizes differently depending on whether it's using the public lookup.  I think we'll need to do the same with test results eventually.